lib/filters/options_processor/options/lib/constructOutput.js

```diff
@@ -0,0 +1,12 @@
+module.exports = (filter, input, rule, result, error) => {
```

Comment:
I feel like there are alot of duplicate code (correct me if i am wrong), what do you think about refactoring the existing code to be reusable?

Reply:
those functions are slightly different, as it's a `filter` and not a `validator`, I rename the object fields.
see if it's better now